src/base/search/searchpluginmanager.cpp

```diff
@@ -172,7 +172,7 @@ void SearchPluginManager::updatePlugin(const QString &name)
 // Install or update plugin from file or url
 void SearchPluginManager::installPlugin(const QString &source)
 {
-    qDebug("Asked to install plugin at %s", qUtf8Printable(source));
+    Logger::instance()->addMessage(tr("Asked to install plugin at %1").arg(source), Log::INFO);
```

Comment:
If you want to print all these messages in a log, you have to make them more meaningful. Look at it through the eyes of the user. It must understand what is at stake.